internal/js/modules/k6/browser/common/browser.go

```diff
@@ -453,8 +453,9 @@ func (b *Browser) onDetachedFromTarget(ev *target.EventDetachedFromTarget) {
 }
 
 func (b *Browser) newPageInContext(id cdp.BrowserContextID) (*Page, error) {
-	if b.context == nil || b.context.id != id {
-		return nil, fmt.Errorf("missing browser context %s, current context is %s", id, b.context.id)
+	bc := b.getDefaultBrowserContextOrMatchedID(id)
+	if bc.id != id {
```

Comment:
Are we sure the default browser context can't be nil? Should we handle this case to avoid future panics?